files/en-us/web/api/displaymediastreamconstraints/video/index.html

```diff
@@ -47,21 +43,50 @@ <h2 id="Syntax">Syntax</h2>
 
 <h3 id="Value">Value</h3>
 
-<p>The value may be either a Boolean or a {{domxref("MediaTrackConstraints")}} object.</p>
+<p>The value may be either a boolean or a "constraints" object.</p>
```

Comment:
```suggestion
<p>The value may be either a boolean value or a "constraints" object.</p>
```

This one is a matter of personal taste, I will not block on it.